src/providers/google/youtube-favourite.ts

```diff
@@ -1,17 +1,25 @@
 import GoogleHandler from "./GoogleHandler";
 import CONFIG from "../../config";
-import { SyncProviderLogEvent, SyncProviderLogLevel } from '../../interfaces'
+import { SyncHandlerPosition, SyncItemsBreak, SyncItemsResult, SyncProviderLogEvent, SyncProviderLogLevel } from '../../interfaces';
 import {
     SyncResponse,
-    SyncHandlerPosition,
     SyncHandlerStatus,
+    HandlerOption,
+    ConnectionOptionType,
 } from "../../interfaces";
 import { SchemaFavouriteContentType, SchemaFavouriteType, SchemaFavourite } from "../../schemas";
 import { google, youtube_v3 } from "googleapis";
 import { GaxiosResponse } from "gaxios";
+import { ItemsRangeTracker } from "../../helpers/itemsRangeTracker";
 
 const _ = require("lodash");
 
+const MAX_BATCH_SIZE = 50;
```

Comment:
Can you confirm why 50 was chosen and add a comment.

Reply:
> Can you confirm why 50 was chosen and add a comment.

The `maxResults` parameter in the Youtube Data API is capped at 50. That's why I set the `MAX_BATCH_SIZE` as 50.